src/test_scripts/vol_test.py

```diff
@@ -59,7 +63,7 @@
     if (os.environ.get('USER_WANT_SPDK')) or ("--spdk" in addln_opts):
         app_mem_size_in_gb = ' --app_mem_size_in_gb=5' # set to 5GB for spdk mode;
 
-vol_addln_opts = addln_opts + emulate_hdd + app_mem_size_in_gb + skip_vol_verify_recovery
+vol_addln_opts = addln_opts + emulate_hdd + app_mem_size_in_gb + skip_vol_verify_recovery + http_port
```

Comment:
I saw you added http port for both log store and metablk store test, but this option is not consumed by test_log_store and test_meta_blk_store, is this expected?

Reply:
Do you mean for `meta_blk_store_nightly` and `logstore_nightly`?

Reply:
yes